.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Feature Requests
+    url: https://community.bitwarden.com/c/feature-requests/
+    about: Request new features using the Community Forums. Please search existing feature requests before making a new one
```

```suggestion
    about: Request new features using the Community Forums. Please search existing feature requests before making a new one.
```

Sorry, this is petty but it will annoy me if I don't fix it now.